Expose a LocationChangeAction type for TypeScript?

Similar to #86. I'd be happy to send a PR for both/either of these issues.

[TypeScript playground example](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgGQPIGECCAVAkqgOQH10AJTAgcQFEAaFCAYwEMZgIA7dAC2Y4HMAppkZtOcAL5wAZlAgg4AIkacOg0YIAmAWjUAPGNrkBXGIKiKA3AFgAUHaA)

```ts
example.ts:1:27 - error TS2305: Module '"connected-next-router"' has no exported member 'LocationChangeAction'.

1 import { LOCATION_CHANGE, LocationChangeAction } from "connected-next-router";
```

The type is exported from `connected-react-router`: https://github.com/supasate/connected-react-router/blob/c6609e335c1b6bd3a2fce163d4d19beeb6d6a7ea/index.d.ts#L41

--- a/src/index.ts
+++ b/src/index.ts
@@ -6,7 +6,13 @@ import _createRouterMiddleware from './middleware'
 
 export {
   LOCATION_CHANGE,
+  LocationChangeAction,
   CALL_ROUTER_METHOD,
+  CallRouterMethodAction,
+  CallRouterMethodPushPayload,
+  CallRouterMethodReplacePayload,
+  CallRouterMethodGoPayload,
+  CallRouterMethodPrefetchPayload,
   routerActions,
   push,
   replace,
@@ -16,6 +22,7 @@ export {
   prefetch,
 } from './actions'
 export { default as routerMethods } from './routerMethods'
+export { LocationState, RouterState } from './types'
 
 export const initialRouterState = createInitialRouterState(plainStructure)
 export const routerReducer = createRouterReducer(plainStructure)

